modules/afsmtp/afsmtp.c

```diff
@@ -225,13 +224,11 @@ afsmtp_dd_format_persist_name(LogThrDestDriver *d)
 /*
  * Worker thread
  */
+
 static void
-afsmtp_dd_msg_add_recipient(AFSMTPRecipient *rcpt, smtp_message_t message)
+afsmtp_dd_msg_add_recipient_header(smtp_message_t message, AFSMTPRecipient *rcpt, AFSMTPDriver *self)
```

Comment:
Please make `AFSMTPDriver *self` parameter as the first parameter. It's a convention to use the `self` parameter as the first one.